sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1932,10 +1932,10 @@ def task_worker(self, stopping_event, sfp_error_event):
                                     notify_media_setting(logical_port, transceiver_dict, self.xcvr_table_helper.get_app_port_tbl(asic_index), self.port_mapping)
                                     transceiver_dict.clear()
                             elif value == sfp_status_helper.SFP_STATUS_REMOVED:
-                                helper_logger.log_info("Got SFP removed event")
+                                helper_logger.log_info("{}: Got SFP removed event".format(logical_port))
```

Comment:
Can we make this and other crucial logs such as SFP insertion event as log_notice as log_info is not captured by default log level?